src/tsbench/config/dataset/_factory.py

```diff
@@ -47,3 +54,50 @@ def get_dataset_config(
     # Get the dataset
     assert name in DATASET_REGISTRY, f"Dataset name '{name}' is unknown."
     return DATASET_REGISTRY[name](Path(path))
+
+def construct_pandas_frame_from_iterable_dataset(
```

Comment:
Is this already supported in autogluon.timeseries?

Reply:
ok, this code was created when I integrated auto pytorch, it is for testing, I will remove it